cleverhans/attacks_tfe.py

```diff
@@ -189,3 +194,100 @@ def fgm(self, x, labels, targeted=False):
         if (self.clip_min is not None) and (self.clip_max is not None):
             adv_x = tf.clip_by_value(adv_x, self.clip_min, self.clip_max)
         return adv_x
+
+
+class BasicIterativeMethodTFE(AttackTFE, BasicIterativeMethod):
```

Comment:
I agree with Ian that it would be preferable not to postfix every name with FTE and possibly use import mechanisms instead to separate different targets (like TF and TFE and ...).